.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!-- Write a description of what the PR solves and how -->
+
+<!-- Link to relevant Jira issue, add multiple if necessary -->
+Jira Issue: [RHELC-]()
+
+Checklist
+- [ ] PR meets acceptance criteria
+- [ ] PR has been tested manually in a VM (either author or reviewer)
+- [ ] Jira issue has been made public if possible

```

Comment:
I don't think this is a relevant check for someone that is opening a pull request… I think that most of us never made any issues public when we pick up something from the board to work on.

Reply:
It is a way to make sure that your task is made public, we want to make as many issues as public as possible